photon-core/src/main/java/org/photonvision/vision/pipeline/OutputStreamPipeline.java

```diff
@@ -126,6 +119,13 @@ public CVPipelineResult process(
             pipeProfileNanos[8] = 0;
         }
 
+        // Draw 2D contours on input and output
+        var draw2dTargetsOnInput = draw2dTargetsPipe.run(Pair.of(inMat, targetsToDraw));
+        sumPipeNanosElapsed += pipeProfileNanos[5] = draw2dTargetsOnInput.nanosElapsed;
```

Comment:
this will technically break pipeline profiling, but as it was already broken it doesn't matter